README.md

```diff
@@ -36,215 +51,26 @@ Minishift is a tool that makes it easy to run OpenShift locally. Minishift runs
 
 * Other OS
 
-  See the installation instructions for the [latest release](https://github.com/jimmidyson/minishift/releases).
+  [TODO - CANNOT FIND INSTRUCTIONS] See the [installation instructions for the latest release](https://github.com/minishift/minishift/releases).
```

Comment:
There are not instructions, so we need to reword this. It should just say for now "Download minishift binary matching your target OS from release page"

There is a bit more to it as well. See  https://github.com/minishift/minishift/issues/171

So really, it should be

* Download binary matching your OS
* Rename to minishift 
  * Technically the rename is not required, but if the docs keep using _minishift_ as binary everywhere, but the user has actually _minishift-linux-amd64_, it might get confusing.
* Make executable
* (Add to PATH)

However, if we change the artifact types on the release page things will get a bit easier. For now I suggest to change to something along "Download minishift binary matching your target OS from release page"